website/content/docs/connect/gateways/mesh-gateway/wan-federation-via-mesh-gateways.mdx

```diff
@@ -126,7 +126,7 @@ connect {
 }
 ```
 
-Any references to [`start_join_wan`](/docs/agent/options#start_join_wan) or [`retry_join_wan`](/docs/agent/options#retry_join_wan) should be omitted.
+The [`start_join_wan`](/docs/agent/options#start_join_wan) or [`retry_join_wan`](/docs/agent/options#retry_join_wan) are only used for the legacy federation process. They must be omitted when federating Consul servers via gateways.
```

Comment:
Why are we calling the non-mesh-gateway federation legacy federation? Isn't that the standard federation model?  Wouldn't this be more correct?

```suggestion
The [`start_join_wan`](/docs/agent/options#start_join_wan) or [`retry_join_wan`](/docs/agent/options#retry_join_wan) must be omitted when federating Consul servers via mesh gateways. 
```